### PR TITLE
add v3 info

### PR DESCRIPTION
**Pull request by vmidyllic**

### CODE REVIEW

File: mkdocs/docs/protocol/main-circuits.md [hunk 1]
```diff
@@ -203,11 +207,64 @@ This circuit checks that an issuer has issued a claim for identity and validates
...
+
+- [**Circuit Specific Files (version 1.0.0-beta.0, NOT Trusted Setup)**](https://iden3-circuits-bucket.s3.eu-west-1.amazonaws.com/latest.zip)
+
+
+This circuit checks that an issuer has issued a claim for identity and validates ownership of that identity in the following manner:
+
+1. Verifies that the identity or identity profile is the subject of the credential. 
+2. Verifies that the schema in the core claim representation contains hash of the credential type identifier.
+3. Verifies that credential is not expired.
+4. Depending on the proof of the verifiable credential (Iden3SparseMerkleTreeProof of BJJSignature) determines the proof verification flow and tree roots to verify.
```
Comment (volodymyr-basiuk) [inline mkdocs/docs/protocol/main-circuits.md:234]: Iden3SparseMerkleTreeProof `or` BJJSignature

Comment (volodymyr-basiuk) [reply mkdocs/docs/protocol/main-circuits.md:234]: and `a` tree roots

File: mkdocs/docs/protocol/main-circuits.md [hunk 2]
```diff
@@ -203,11 +207,64 @@ This circuit checks that an issuer has issued a claim for identity and validates
...
+
+
+- [**Github**](https://github.com/iden3/circuits/blob/develop/circuits/onchain/credentialAtomicQueryV3OnChain.circom)
+
+- [**Example of instantiation**](https://github.com/iden3/circuits/blob/develop/circuits/credentialAtomicQueryV3OnChain.circom)
+
+- [**Circuit Specific Files ( version 1.0.0-beta.0, NOT Trusted Setup)**](https://iden3-circuits-bucket.s3.eu-west-1.amazonaws.com/latest.zip)
+
+
+This circuit should be used for smart contract verifiers. This circuits does all the checks that the the credentialAtomicQueryV3 circuit does, plus the following:
```
Comment (volodymyr-basiuk) [inline mkdocs/docs/protocol/main-circuits.md:265]: `the the`, remove one please

File: mkdocs/docs/protocol/main-circuits.md [hunk 3]
```diff
@@ -203,11 +207,64 @@ This circuit checks that an issuer has issued a claim for identity and validates
...
+
+1. Checks that prover controls the identity in the same way as the AuthV2 circuit checks it in case auth is enabled.
```
Comment (volodymyr-basiuk) [inline mkdocs/docs/protocol/main-circuits.md:267]: better ...`in the same way as the AuthV2 if auth is enabled.`

File: mkdocs/docs/protocol/main-circuits.md [hunk 4]
```diff
@@ -203,11 +207,64 @@ This circuit checks that an issuer has issued a claim for identity and validates
...
+2. Verifies credential query in the same way as credentialAtomicQueryV3 does.
+3. Calculates hash of the query inputs, like claimSchema, slotIndex, operator, claimPathKey, claimPathNotExists and values as an output for all the query related inputs.
+   This reduces the number of public inputs and much cheaper for Smart Contracts to verify the proof.
```
Comment (volodymyr-basiuk) [inline mkdocs/docs/protocol/main-circuits.md:270]: maybe better ...` for Smart Contracts verification.`

**Review by volodymyr-basiuk — APPROVED** [review]

**Review by OBrezhniev — DISMISSED** [review]

**Review by volodymyr-basiuk — APPROVED** [review]

**Pull request merged**